tests/rbd/rbd_utils.py

```diff
@@ -730,3 +763,41 @@ def rbd_remove_image_negative_validate(rbd, pool, image):
         log.error(f"{out}")
         raise ImageIsDeletedError(f" RBD image is deleted: {out}")
     log.info(f"{err}")
+
+
+def verify_migration(rbd, dest_spec):
+    """
+    This method will verify for the Live migration is prepared or not
+    Args:
+        rbd: rbd object
+        dest_spec: Target image spec in format of TARGET_POOL_NAME/SOURCE_IMAGE_NAME
+    """
+    log.info("verify migration process started")
+    out = rbd.exec_cmd(cmd=f"rbd status {dest_spec} --format json", output=True)
+    log.info(out)
+    status = json.loads(out)
+    if "prepared" in status["migration"]["state"]:
+        log.info("Live Migration successfully prepared")
+        return 0
+    elif "executed" in status["migration"]["state"]:
+        log.info("Live migratin successfully executed")
```

Comment:
Can we please add dest_spec in log.infos so that when we execute this on scale, it helps us to use log properly

Reply:
Done.